### PR TITLE
Add Focal copy number calls methods

### PR DESCRIPTION
**Pull request by cbethell**
### Purpose

To add a methods section on how the focal copy number calls were made(well are being made). 
I used what is in the [Focal CN File Preparation README](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/focal-cn-file-preparation) and the relevant PRs and tickets for reference.

### Issue

Not one issue in particular.

#### Which areas should receive a particularly close look?

I am not sure that I covered all of the necessary details from the analysis in the methods section here, so someone who is familiar with the `focal-cn-file-preparation` module should probably give this a first look.

#### Is the pull request ready for review?
Yes?

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

**Comment by cgreene** [comment]
ok - updated this one - hoping the build works with the new system 🤞

**Comment by cgreene** [comment]
looking promising! github actions / appveyor both going

File: content/03.methods.md [hunk 1]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
 CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
 We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh).
 
+#### Focal Copy Number Calling
+
+We added the ploidy inferred via ControlFreeC to the CNVkit and consensus CNV segmentation files and used the ploidy and copy number values to define a status column with gain and loss values broadly.
```
Comment (cgreene) [inline content/03.methods.md:149]:
```suggestion
We added the ploidy inferred via Control-FREEC to the CNVkit and consensus CNV segmentation files and used the ploidy and copy number values to define a status column with gain and loss values broadly.
```
Noticed this one when I was adding the spell checker

**Comment by cgreene** [comment]
I'd suggest adding `cytoband` to the dictionary. I'm not sure what to do with `bedtools`. @arq5x's paper says `BEDtools` ( https://academic.oup.com/bioinformatics/article/26/6/841/244688 ) but readthedocs for the project uses `bedtools`.

**Comment by arq5x** [comment]
re: bedtools, I tend to use all lower case now, but I could care less.

File: content/03.methods.md [hunk 2]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
...
+We used bedtools coverage [@url:https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html] to add cytoband status using the UCSC cytoband file [@url:http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz].
+The output status call fractions were used to define dominant status at the cytoband-level.
+The weighted means of each status call fraction and band length were calculated and used to define the dominant status at the chromosome arm-level.
```
Comment (jaclyn-taroni) [inline content/03.methods.md:152]: In addition to the URL citations here, I would cite the papers associated with these tools/resources. For example: https://github.com/arq5x/bedtools2#citation

File: content/03.methods.md [hunk 3]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
 CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
 We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh).
 
+#### Focal Copy Number Calling
+
+We added the ploidy inferred via ControlFreeC to the CNVkit and consensus CNV segmentation files and used the ploidy and copy number values to define a status column with gain and loss values broadly.
+We used bedtools coverage [@url:https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html] to add cytoband status using the UCSC cytoband file [@url:http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz].
+The output status call fractions were used to define dominant status at the cytoband-level.
```
Comment (jaclyn-taroni) [inline content/03.methods.md:151]: I would be very explicit about what "output status call fraction" means.

File: content/03.methods.md [hunk 4]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
...
+The weighted means of each status call fraction and band length were calculated and used to define the dominant status at the chromosome arm-level.
+
+A status is considered dominant if the more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
+The 0.9 threshold was chosen to ensure that the dominant status fraction call is indeed significantly greater than the remaining status fraction calls in a region.
```
Comment (jaclyn-taroni) [inline content/03.methods.md:155]:
```suggestion

```
I don't know if you need this sentence. At the very least I would remove the phrase "indeed significantly greater."

File: content/03.methods.md [hunk 5]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
...
+
+For the determination of the most focal units, we first considered the dominant status calls at the chromosome arm-level.
```
Comment (jaclyn-taroni) [inline content/03.methods.md:157]: Why do you need to call most focal units? Contextualize this.

File: content/03.methods.md (hunk 2 above)
Comment (jaclyn-taroni) [inline content/03.methods.md:152]: I find this sentence about weighted means confusing - you used the band length in the weighted mean calculation, correct?

File: content/03.methods.md (hunk 1 above)
Comment (jaclyn-taroni) [reply content/03.methods.md:149]: You don't need to say status _column_. You do need to be specific about what "gain and loss values broadly" means.

File: content/03.methods.md [hunk 6]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
...
+If the chromosome arm was not clearly defined as a gain or loss (and was callable) we looked to include the cytoband-level status call.
```
Comment (jaclyn-taroni) [inline content/03.methods.md:158]: "Clearly defined" just refers to the dominant status calculation above, correct?

File: content/03.methods.md [hunk 7]
```diff
@@ -144,6 +144,20 @@ Theta2 purity was added as an optional parameter to CNVkit to adjust copy number
...
+Similarly, if a cytoband was not clearly defined as a gain or loss (and was callable) we looked to include the gene-level status call.
```
Comment (jaclyn-taroni) [inline content/03.methods.md:159]: Is there information about how we got the gene-level status call anywhere?

File: content/03.methods.md (hunk 6 above)
Comment (cbethell) [reply content/03.methods.md:158]: That's correct.

File: content/03.methods.md (hunk 2 above)
Comment (cbethell) [reply content/03.methods.md:152]: Correct, will clarify this.

File: content/03.methods.md (hunk 7 above)
Comment (cbethell) [reply content/03.methods.md]: Added some information around this in the latest commits.

**Pull request merged**